front/app/component-library/components/Icon/index.tsx

```diff
@@ -425,6 +425,139 @@ export const icons = {
       </defs>
     </Svg>
   ),
+  idaustria: (props: IconPropsWithoutName) => (
```

Comment:
This seems quite a big SVG considering it is only used on one platform, but I don't know what we can do about that